addons/sale_stock/models/sale_order.py

```diff
@@ -238,6 +238,17 @@ def _get_qty_procurement(self):
                 qty -= move.product_uom._compute_quantity(move.product_uom_qty, self.product_uom, rounding_method='HALF-UP')
         return qty
 
+    def _get_procurement_group(self):
+        return self.order_id.procurement_group_id
+
+    def _prepare_procuremnt_group(self):
```

Comment:
s/procuremnt/procurement and put at the end _vals, which is the usual method name.